addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -92,6 +92,10 @@ Number Netatmo_Indoor_CO2 "CO2" <carbondioxide> { channel = "netatmo:NAMain:home
 * Location
 * TimeStamp
 * LastStatusStore
+* MinTemp
+* MaxTemp
```

Comment:
We have a confirmateion that the NAModule4 supports the new channels https://community.openhab.org/t/what-channel-types-are-supported-by-wich-netatmo-device/24394/24.